tpgtools/overrides/monitoring/samples/monitoredproject/meta.yaml

```diff
@@ -0,0 +1,7 @@
+substitutions:
```

Comment:
These feel like things we shouldn't need to define substitutes for- can you file an issue to improve our default choices? (fyi @ScottSuarez / @c2thorn)

Reply:
b/200981288.